src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -54,13 +60,14 @@ export const EditRespondentPopup = ({
   const submitForm = () => {
     if (!chosenAppletData) return;
 
-    const { secretUserId, nickname } = getValues();
+    const { secretUserId, nickname, tag } = getValues();
     const { subjectId } = chosenAppletData;
 
     editRespondent({
       values: {
         secretUserId: secretUserId.trim(),
         nickname: nickname?.trim(),
+        ...(tag ? { tag } : {}),
```

Comment:
Does this work, or is the ternary format a better practice?
```suggestion
        ...(tag && { tag }),
```

Reply:
The statements are approximately equivalent in effect. False-y values act like an empty object if you apply the object spread operator to them, so the ternary isn't really necessary, unless you favour the explicitness in how its read. 

I don't have any particular opinion — I've updated it to your suggestion for conciseness.